docs/en/inference/pipeline.md

```diff
@@ -224,6 +224,7 @@ This class provides the configuration parameters for Pytorch backend.
 | num_cpu_blocks        | int   | Number of CPU blocks. If the number is 0, cache would be allocated according to the current environment.                              | 0           |
 | num_gpu_blocks        | int   | Number of GPU blocks. If the number is 0, cache would be allocated according to the current environment.                              | 0           |
 | adapters              | dict  | The path configs to lora adapters.                                                                                                    | None        |
+| thread_safe           | dict  | EngineInstance is threadsafe or not.                                                                                                  | False       |
```

Comment:
#1203 will remove this section in `pipeline.md`
So, may rollback changes to the pipeline documents.